src/main/java/org/researchspace/data/rdf/container/LDPAssetsLoader.java

```diff
@@ -150,14 +203,55 @@ public void load() throws StorageException, IOException {
 
             currentStorageMap.put(entry.getKey(), entry.getValue());
         }
-
+ 
         // Load each batch separately into the corresponding repository
         for (Entry<String, Map<String, Map<StoragePath, FindResult>>> entry : mapResultsByRepositoryIdAndStorageId
                 .entrySet()) {
-            if (isLoadableFromStorage(entry.getKey())) {
+            if (isLoadableFromStorage(entry.getKey()) || defaultRepositoriesList.contains(entry.getKey())) {
                 boolean isWritable = repositoryManager.getRepository(entry.getKey()).isWritable();
-                if (isWritable) {
-                    loadAllToRepository(entry.getKey(), entry.getValue());
+                if (isWritable) { 
+                                                 
+                    if ((loadDefaultConfig == 1) && defaultRepositoriesList.contains(entry.getKey())) {
```

Comment:
that is why tests are failing, if loadDefaultConfig is not set then it will not load not only default config but all existing LDP assets

Reply:
not failing after my last set of commits.